### DISCUSSION TITLE
Ubuntu 23.10

### DISCUSSION THREAD

**Discussion by PatriceGendreau** [regolith-linux/regolith-desktop#941] [General]
Just upgrade to Ubuntu 23.10
Regolith works but...
- cannot launch the gnome-terminal, other "terminal" like Kilix work
- win-c inoperate....
-

**Comment by pmahnke** [comment]
@PatriceGendreau are you using:

```
echo deb "[arch=amd64 signed-by=/usr/share/keyrings/regolith-archive-keyring.gpg] \
https://regolith-desktop.org/testing-ubuntu-mantic-amd64 mantic main" | \
sudo tee /etc/apt/sources.list.d/regolith.list
```

in `/etc/apt/sources.list.d/regolith.list`? Because both of these work for me.

**Reply by PatriceGendreau** [reply]
I've done a "clean install" of Ubuntu 23.10 and then install Regolith with the source above
Everythink works better

BUT
I'm on a french keyboard.
I cannot access to a workspace by "win+1" or 2 3 4 but with "Win+shift+1" or 2 3 4
So I cannot "shift" a window to another workspace

I think I have to modify :
/usr/share/regolith/common/config.d/30_navigation
But how ? with amperand or so ?

BUT

I cannot install "looks" 
Problem with : "gnome-theme-standard"

Merci 
(sorry for my poor english, I'm french....)

**Reply by pmahnke** [reply]
I think you would need to copy that config to your local one and edit it there:

`cp /usr/share/regolith/common/config.d/30_navigation ~/.config/regolith3/common-wm/config.d `

and how are you installing the looks?  for example:

You need to install it and then set it:

```
sudo apt install regolith-look-solarized-dark
regolith-look set solarized-dark
regolith-look refresh
```

p.s. your English is great!

**Reply by garte** [reply]
Using your PPA url seems to work for me. Where did you find it? I haven't got a clue how to view the source codebase behind `https://regolith-desktop.org/testing-ubuntu-mantic-amd64`.

I'd like to know more about where this is coming from.

**Reply by garte** [reply]
I guess I figured it out myself. The packages contained in the testing release are configured here: https://github.com/regolith-linux/voulage/blob/main/stage/testing/ubuntu/mantic/package-model.json

**Comment by kgilmer** [comment]
FYI, at this time we do not have official support for Ubuntu 23.10.  We are working through some issues related to new functionality in GNOME 45.

**Reply by abinadi** [reply]
Any updates on this?

**Reply by kgilmer** [reply]
Hi @abinadi , regarding the interop issue that's blocking the release, no simple [solution has been found](https://gitlab.gnome.org/GNOME/xdg-desktop-portal-gnome/-/issues/104#note_1926837). So a xdg-portals backend is being [developed for Regolith](https://github.com/regolith-linux/xdg-desktop-portal-regolith).  Once that's ready to go we can proceed w/ release testing.

**Reply by stephenh** [reply]
I was just wandering by to check in, and noticed all of the super-recent activity from @kgilmer investigating the root cause/workarounds; looking forward to a resolution and really appreciate it!

**Reply by kgilmer** [reply]
Hi @stephenh , thanks!  Yes in testing it seems that the sort of "empty" portal implementation enables us to side-step issues with the GNOME portal implementation.  Moving forward with a beta release, stay tuned.

**Reply by kgilmer** [reply]
Regolith 3.1 Beta 1 is ready for testing. Details here: https://regolith-desktop.com/docs/reference/Releases/regolith-3.1-release-notes/

**Reply by stephenh** [reply]
Awesome! I probably won't get a chance to test it right away, but just checking, should I upgrade to Regolith 3.1 first, then Ubuntu 23.10?

**Reply by kgilmer** [reply]
I suggest upgrading Ubuntu first, then installing regolith.

**Reply by stephenh** [reply]
Looks like the testing release isn't available anymore:

```
E: The repository 'https://regolith-desktop.org/testing-ubuntu-manic-amd64 manic Release' no longer has a Release file.
```

But happy for that to be because the non-testing release is out/coming soon. :-)

**Reply by garte** [reply]
You might have a typo in your definition: It's `mantic` not `manic`?

I'm not getting an error message when updating, that update channel still works for me.

**Reply by kgilmer** [reply]
Yep, you've been bitten by my typo @stephenh .  I recently deleted the repos with "manic" in the URL which is why you now see the error.  Update the apt line to "mantic" and you should be back to consuming Regolith from the `testing` stage.

**Reply by stephenh** [reply]
Oh great! That works!

Good to know, I would have gone around saying "hey guys, I'm on Ubuntu Manic and it's going great", TIL the release name is __mantic__. :-D

**Comment by stephenh** [comment]
I got one error during the regolith upgrade:

```
(Reading database ... 300075 files and directories currently installed.)
Preparing to unpack .../sway-regolith_1.8.1-3-1regolith-manic_amd64.deb ...
Unpacking sway-regolith (1.8.1-3-1regolith-manic) over (1.7-6-1regolith-lunar) ...
dpkg: error processing archive /var/cache/apt/archives/sway-regolith_1.8.1-3-1regolith-manic_amd64.deb (--unpack):
 trying to overwrite '/usr/share/backgrounds/sway/Sway_Wallpaper_Blue_1136x640.png', which is also in package sway-backgrounds 1.8.1-2
```

So just had to first apt remove `sway-backgrounds` first, and then everything resolved/rebooted fine! So far so good! :crossed_fingers: Thank you!

**Comment by bitsonymous** [comment]
How to Resolve Repository Errors in Ubuntu Mantic (No Release File, Network Issues)?

Description of the Issue:

I've recently upgraded to Ubuntu Mantic, but I keep running into issues with repositories. Some repositories fail to fetch packages, and I receive errors like "No Release file," "Network is unreachable," or "Repository is not signed." Here are some examples of the error messages I’m encountering:

`E: The repository 'https://ppa.launchpadcontent.net/flatpak/stable/ubuntu mantic Release' does not have a Release file.
``E: Failed to fetch https://packages.redis.io/deb/dists/mantic/InRelease 403 Forbidden
`Err: Cannot initiate the connection to ubuntu-ports.mirror.net.in:443 (101: Network unreachable)
`W: Key is stored in legacy trusted.gpg keyring...
`I've tried updating my package lists with sudo apt update, but the issues persist. I've also checked my 

`/etc/apt/sources.list, and there are some repositories related to mantic which seem to be causing problems.`

Steps Taken:

- Tried commenting out the problematic repositories in the sources.list.
- Attempted to switch to other mirrors, but still facing connection issues.
- I noticed that i386 architecture is mentioned even though I’m on an amd64 system.
- Attempted to update repository keys using curl for some repositories like PostgreSQL, but it didn’t solve the issue.
